schema/bom-1.7.xsd

```diff
@@ -3931,7 +3931,7 @@ limitations under the License.
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
-            <xs:element name="sensitiveData" minOccurs="0" maxOccurs="unbounded">
+            <xs:element name="sensitiveData" type="xs:string" minOccurs="0" maxOccurs="unbounded">
```

Comment:
in JSON its an array of strings.
in protobug it is repeated string.

conclusion: was intended to be unbound string in XML, too